dev/com.ibm.ws.messaging.open_jms20context_fat/publish/files/unused/185047_Bindings.xml

```diff
@@ -0,0 +1,76 @@
+<server>
+    <featureManager>                                                   
+        <feature>jsp-2.3</feature>                                     
+                <feature>wasJmsServer-1.0</feature>                    
+                <feature>wasJmsClient-2.0</feature>                    
+                <feature>jmsMdb-3.2</feature>
+                <feature>testjmsinternals-1.0</feature>
+                 <feature>timedexit-1.0</feature>                           
+    </featureManager>
+
+    <include optional="true" location="../fatTestPorts.xml"/>
+
+    <variable name="onError" value="FAIL"/>
+
+    <messagingEngine id="defaultME" >
+
+		<queue id="Queue1" 
+			   maxRedeliveryCount="2"
+			   >
+		</queue>
+		<queue id="Queue2" 
+			   maxRedeliveryCount="2"
+			   >
+		</queue>	
+	
+		
+    <fileStore id="com.ibm.ws.sib.fileStore"
+      logFileSize="10">
+    </fileStore>
+
+	 </messagingEngine>
+   
+ <jmsQueue id="jndi_INPUT_Q" jndiName="jndi_INPUT_Q">
+      <properties.wasJms queueName="newQueue" />
+    </jmsQueue> 
+          
+    <jmsQueue id="eis/queue1" jndiName="eis/queue1">
+      <properties.wasJms queueName="QUEUE1" />
+    </jmsQueue>
+    
+    <jmsQueue id="eis/queue2" jndiName="eis/queue2">
+      <properties.wasJms queueName="Queue2" />
+    </jmsQueue>
+    
+    <jmsQueue id="queue/test" jndiName="queue/test">
+      <properties.wasJms queueName="Queue10" />
+    </jmsQueue>
+    
+     <jmsQueue id="Queue11/test" jndiName="Queue11/test">
+      <properties.wasJms queueName="Queue11" />
+    </jmsQueue>
+    
+    
+    <connectionManager id="ConMgr5" maxPoolSize="2"/>
+    
+   <jmsQueueConnectionFactory jndiName="jndi_JMS_BASE_QCF1" connectionManagerRef="ConMgr5">                    
+        <properties.wasJms remoteServerAddress="localhost:${bvt.prop.jms.1}:BootstrapBasicMessaging"           
+targetTransportChain="InboundBasicMessaging"/>                         
+    </jmsQueueConnectionFactory>
+    
+     <jmsQueueConnectionFactory jndiName="jndi_JMS_BASE_QCF" connectionManagerRef="ConMgr6">
+      <properties.wasJms userName="CF1USER" password="junkpassword" /> 
+												
+    </jmsQueueConnectionFactory>
+    
+      
+    
+    <jmsActivationSpec id="jmsapp/jmsmdb/AckModeMessageDrivenBean">
```

Comment:
Eventually this server configuration file, and all others that contain jmsActivationSpec elements that declare a javax.jms.Queue or Topic as the destinationType, must be transformed.  Server and client xml files that require transformation for messaging tests are declared in the jakarta-xml-master.properties file in section `#Messaging`.

Reply:
Noted.  Very probably, the MDB application is not used.